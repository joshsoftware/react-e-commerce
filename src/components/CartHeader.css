.header {
    background-color:#f2f2f2;
    padding: 0px 15px;
    text-align: center;
    margin-top: 120px;
}

h1 {
    padding: auto;
}