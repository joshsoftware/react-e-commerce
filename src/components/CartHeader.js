import React from 'react';
import './CartHeader.css';
import PropTypes from 'prop-types';
import ContainerWrapper from './ContainerWrapper';

const CartHeader = ({ header }) => {
  return <ContainerWrapper styleClass="header" data={<h1>{header}</h1>} />;
};

export default CartHeader;

CartHeader.propTypes = {
  header: PropTypes.string
};
