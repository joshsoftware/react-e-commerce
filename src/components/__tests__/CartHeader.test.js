import React from 'react';
import CartHeader from '../CartHeader';
import { render } from '@testing-library/react';

describe('CartHeader Component', () => {
  it('Must return cart header', () => {
    const { asFragment } = render(<CartHeader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
