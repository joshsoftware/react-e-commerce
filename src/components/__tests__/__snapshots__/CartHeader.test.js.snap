// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartHeader Component Must return cart header 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1>
      Cart Items
    </h1>
  </div>
</DocumentFragment>
`;
